assets/js/components/ResetButton.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * ResetButton component.
+ *
+ * Site Kit by Google, Copyright 2020 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * WordPress dependencies
+ */
+import { __ } from '@wordpress/i18n';
+import { Fragment, useState, useEffect, useCallback, createInterpolateElement } from '@wordpress/element';
+
+/**
+ * Internal dependencies
+ */
+import Data from 'googlesitekit-data';
+import { clearWebStorage } from '../util';
+import Dialog from './dialog';
+import Modal from './Modal';
+import Link from './Link';
+import { STORE_NAME as CORE_SITE } from '../googlesitekit/datastore/site/constants';
+
+const { useSelect, useDispatch } = Data;
+
+function ResetButton( { children } ) {
+	const postResetURL = useSelect( ( select ) => select( CORE_SITE ).getAdminURL( 'googlesitekit-splash', { notification: 'reset_success' } ) );
+
+	const [ dialogActive, setDialogActive ] = useState( false );
+	// const postResetURL = addQueryArgs( splashURL,  );
+
+	const handleCloseModal = useCallback( ( event ) => {
+		if ( 27 === event.keyCode ) {
+			// Only close the modal if the "Escape" key is pressed.
+			setDialogActive( false );
+		}
+	} );
+
+	useEffect( () => {
+		if ( dialogActive ) {
+			// When the dialogActive changes and it is set to true(has opened), add the event listener.
+			global.addEventListener( 'keyup', handleCloseModal, false );
+		}
+		// Remove the event listener when the dialog is removed; there's no need
+		// to have it attached when it won't be used.
+		return () => {
+			if ( dialogActive ) {
+				// When the dialogActive is true(is open) and its value changes, remove the event listener.
+				global.removeEventListener( 'keyup', handleCloseModal );
+			}
+		};
+	}, [ dialogActive ] );
+
+	const { reset } = useDispatch( CORE_SITE );
+
+	const handleUnlinkConfirm = useCallback( async () => {
+		await reset();
+		clearWebStorage();
+		setDialogActive( false );
+		global.location.assign( postResetURL );
+	}, [ reset, postResetURL ] );
+
+	const toggleDialogActive = useCallback( () => {
+		setDialogActive( ! dialogActive );
+	}, [ dialogActive ] );
+
+	return (
+		<Fragment>
+			<Link
```

Comment:
Not sure why this was a `Link` before, but let's make it a `<Button>` instead, including `type="button"` so it doesn't potentially trigger a form submit when clicked.

Reply:
`Button`:
 ![image](https://user-images.githubusercontent.com/72751210/98671548-71f81c80-237a-11eb-980a-2cb1ea5494cc.png)

`Link`(This is how it is right now):
 ![image](https://user-images.githubusercontent.com/72751210/98671593-83d9bf80-237a-11eb-8a01-760b705a60ad.png)

Do you want me to change this?

Reply:
Oh, no you're right 😄  my mistake, I was thinking of the button that is shown in the modal when you click that 👍 

Please keep as-is, thank you 🙂